test/rcedit-test.js

```diff
@@ -75,14 +101,14 @@ describe('async rcedit(exePath, options)', function () {
     await rcedit(exePath, { 'product-version': '1' })
 
     const info = await rcinfo(exePath)
-    assert.strictEqual(info.ProductVersion, '1.0.0.0')
+    assert.strictEqual(info.ProductVersion, '1')
```

Comment:
Is this considered a breaking change?

Reply:
Nope - the change here is just that `rcinfo` read the normalized product version, but my refactor using `rcedit` doesn't have that capability. It's all within the test and has no impact on the shipping code.